blackbox/docs/general/ddl/partitioned-tables.rst

```diff
@@ -372,9 +372,13 @@ both existing and future partitions of the table.
     cr> ALTER TABLE parted_table ADD COLUMN new_col string
     ALTER OK, -1 rows affected (... sec)
 
-Unlike regular tables, it is possible to change the number of shards of a
-partitioned table. However, the shard configuration will only be applied on
-**future** partitions.
+
+Changing the Number of Shards
+-----------------------------
+
+It is possible at any time to change the number of shards
+of a partitioned table. However, the shard configuration
+will only be applied on -**future** partitions.
```

Comment:
"be applied to" instead of "on"

also, there's an extra "-" character before **future**. I'd also maybe replace **future** with **new** partitions? (we use "new" below as well and I think it makes more sense)

Reply:
+1 for "new" partitions